homework_7/functional.py

```diff
@@ -0,0 +1,92 @@
+import sys
+
+
+def sequential_map(*args):
+    """
+    Accepts any number of functions as positional arguments and a container (list/set/tuple) with values
+    as the last positional argument.
+    Returns a list of the results of sequentially applying the passed functions to the values in the container.
+    """
+    cont = args[len(args) - 1]
+    funcs = args[: len(args) - 1]
```

Comment:
Мне нравится вот так
```suggestion
    *funcs, cont = args
```